nvm.sh

```diff
@@ -4000,6 +4013,7 @@ nvm() {
     ;;
     "ls-remote" | "list-remote")
       local NVM_LTS
+      local NVM_MIN="${NVM_MIN-}"
```

Comment:
```suggestion
      local NVM_MIN
      NVM_MIN="${NVM_MIN-}"
```
this is needed for ksh

Reply:
Ah, good to know!

Nevertheless the idea is to inherit `$NVM_MIN` from env, I'm not sure if that would still work with a change like this, maybe we need to introduce another variable as a workaround?